doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -299,7 +299,7 @@ described elsewhere
 
          Search is:Instance [ Reflexive | Symmetric ].
 
-.. cmd:: SearchPattern @one_pattern {? {| inside | outside } {+ @qualid } }
+.. cmd:: SearchPattern @one_pattern {? {| {| inside | outside } {+ @qualid } | in {+ @qualid } } }
```

Comment:
Please update the following to include "in":

![image](https://user-images.githubusercontent.com/1253341/140803801-9c3ad2e2-5a3f-41f2-bade-28f21a37a536.png)